src/libraries/Microsoft.Extensions.Primitives/tests/CompositeChangeTokenTest.cs

```diff
@@ -103,7 +103,7 @@ public void ActiveChangeCallbacks_IsFalse_IfNoTokenHasActiveChangeCallbacks()
             Assert.False(compositeChangeToken.ActiveChangeCallbacks);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [Fact]
         public async Task RegisteredCallbackGetsInvokedExactlyOnce_WhenMultipleConcurrentChangeEventsOccur()
```

Comment:
same here, it also fails with a threading exception.

Reply:
Done.